Backend/DataAccessLayer/Migrations/201812100518074_user-add-manager.resx

```diff
@@ -0,0 +1,126 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<root>
+  <!-- 
+    Microsoft ResX Schema 
+    
+    Version 2.0
+    
```

Comment:
Up to you, but do you think we should remove these comments?

Reply:
This is an autogenerated file, so my gut instinct is either to leave it as-is or remove (ignore) it entirely.